speech-to-text-template/src/index.ts

```diff
@@ -0,0 +1,15 @@
+export default {
+	async fetch(request, env, ctx): Promise<Response> {
+		const audioResponse = await fetch(
+			"https://github.com/Azure-Samples/cognitive-services-speech-sdk/raw/master/samples/cpp/windows/console/samples/enrollment_audio_katie.wav",
```

Comment:
This template might be more suitable for the gallery if it had a frontend / allowed for variable audio input. Without that, I might prefer to keep it out of our standard set of examples.